networkmanager/networkmanager.go

```diff
@@ -720,16 +774,15 @@ func getBandwidthPluginConfig(ingressKbit, egressKbit uint64) (config json.RawMe
 	}
 
 	// the burst argument was selected relative to the mtu network interface
-	burst := uint64(12800) // bits == 1600 byte
 
 	if ingressKbit > 0 {
-		bandwidth.IngressRate = ingressKbit * 1000
-		bandwidth.IngressBurst = burst
+		bandwidth.IngressRate = ingressKbit * 1000 // nolint:gomnd
```

Comment:
Add 1000 as linter exception to the linter config.

Reply:
Done